datalad/customremotes/tests/test_datalad.py

```diff
@@ -30,14 +31,19 @@ def check_basic_scenario(url, d):
     # TODO skip if no boto or no credentials
     get_test_providers(url) # so to skip if unknown creds
 
+    # git-annex got a fix where it stopped replacing - in the middle of the filename
+    filename = '3versions%sallversioned.txt' % (
+        '_' if external_versions['cmd:annex'] < '8.20200501+git53-gcabbc91b1' else '-'
```

Comment:
This is fine, though I think it'd be even better to check whether a file with one of the two names exists after the `add_urls` call.  The current approach means this test would fail for me with a local git-annex build after 8.20200501 but before that commit, where cmd:annex would be something like `8.20200502-g[xdigit]+`

Reply:
Heh.. I thought to use glob first... Ok, will adjust later while also checking that expected name is popping out for versions outside of the development range

Reply:
Since I do want to have at least this single test to check for expected annex behavior